app/containers/HomeContainer/index.js

```diff
@@ -97,13 +97,20 @@ export function HomeContainer({
                 <T id="matching_repos" values={{ totalCount }} />
               </div>
             )}
-            {items.map((item, index) => (
-              <CustomCard key={index}>
-                <div>Repository Name: {item.name}</div>
-                <div>Repository Full Name: {item.fullName}</div>
-                <div>Repository stars: {item.stargazersCount}</div>
-              </CustomCard>
-            ))}
+            {items.map((item, index) => {
+              return (
+                <Link
+                  key={index}
+                  to={`/${item.owner.login}/${item.name}/issues`}
+                >
+                  <CustomCard key={index}>
+                    <div>Repository Name: {item.name}</div>
```

Comment:
let's add these strings to the en.json and use react-intl  so that we don't lose support loose localization